portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +32,50 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
These Strings like "Comment", "timestamp", "text" get used in a few places. I think it would be better if you kept them as `private static final String` constants at the top of this class. Then if you want to change one you only have to change one place instead of hunting around looking for all of them.

Another option might be to store them also as static fields in your Comment class and then expose them through static methods, or just make the fields public. Either way is fine for me.

Also can you stick to a consistent capitalization strategy? "Comment" capitalizes the first letter but "timestamp" doesn't.

Reply:
Done.